CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+2.0.0/ UNRELEASED
+=================
+
+### Changes
+* [OTHER] Namespace the statsd Class under the Datadog module.
+
```

Comment:
I believe we should make a highlight ➕ of that . `[OTHER]` sounds too much of a small change.